src/lib/hooks/swap/useBestTrade.ts

```diff
@@ -100,9 +41,8 @@ export function useBestTrade(
   return useMemo(
     () => ({
       ...(useFallback ? fallbackTrade : clientSORTrade),
-      ...(syncing ? { state: TradeState.SYNCING } : {}),
```

Comment:
I have a feeling removing the syncing check is what is causing the break on the UI dim state

Reply:
I updated useClientSideSmartOrderRouterTrade to include the trade as a consequence of its fetched state, so the trade will no longer update until a new one is ready (or none is found). I think retaining the trade with the loading state is cleaner than re-inferring it one level down.